src/lib/selector.c

```diff
@@ -308,8 +319,28 @@ int ncselector_delitem(ncselector* n, const char* item){
         }
       }
       --n->itemcount;
-      return ncselector_draw(n);
+      found = true;
+      --idx;
+    }else{
+      int cols = mbswidth(n->items[idx].option);
+      if(cols > maxop){
+        maxop = cols;
+      }
+      cols = mbswidth(n->items[idx].desc);
+      if(cols > maxdesc){
+        maxdesc = cols;
+      }
+    }
+  }
+  if(found){
+    n->longop = maxop;
+    n->longdesc = maxdesc;
+    int dimy, dimx;
+    ncselector_dim_yx(n->ncp->nc, n, &dimy, &dimx);
+    if(origdimx > dimx || origdimy > dimy){ // resize if too big
+      ncplane_resize(n->ncp, 0, 0, 0, 0, 0, 0, dimy, dimx);
```

Comment:
You can replace this with `ncplane_resize_simple(n->ncp, dimy, dimx)` (your other one, too).

Reply:
Actually, that would keep as much of the plane as is possible, which would be unnecessary copying.

Reply:
Oh well, I see you already changed it. I really doubt there's any measurable difference, and this looks a lot cleaner.